rawx/chunk_info.go

```diff
@@ -216,16 +217,20 @@ func (chunk *chunkInfo) loadAttr(inChunk fileReader, chunkID string,
 	chunk.ChunkID = chunkID
 
 	for _, hs := range detailedAttrs {
-		value, err := getAttr(hs.key)
+		*(hs.ptr), err = getAttr(hs.key)
```

Comment:
In case of error, `*(hs.ptr)` will be initialized with an empty string instead of staying uninitialized. Is it wanted?

Reply:
Please not that getAttr always return empty string in case of error.
This behavior was already the case before this PR

Reply:
> Please not that getAttr always return empty string in case of error.
> This behavior was already the case before this PR

Yes, but what about `*(hs.ptr)`?

Reply:
`ptr` reference chunk fields that was previously initialized empty by ServeHTTP method (rawx.go l.118 ) when rawRequest is instancied